src/platforms/web/runtime/components/transition.js

```diff
@@ -114,7 +114,8 @@ export default {
       mode && mode !== 'in-out' && mode !== 'out-in'
     ) {
       warn(
-        'invalid <transition> mode: ' + mode,
+        'invalid <transition> mode: ' + mode +
+        '<transition> mode should be either \'in-out\' or \'out-in\'',
```

Comment:
it can also be an empty string. Let's use `"` instead of `'` like in the rest of the code

```suggestion
        '<transition> mode should be either "in-out", "out-in\" or "" (empty string)',
```

Reply:
@posva thanks